sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -215,7 +218,8 @@ def get(self, name: str, version: Optional[str] = None, label: Optional[str] = N
             log_and_raise_error(ex)
 
     # @monitor_with_activity(logger, "Data.CreateOrUpdate", ActivityType.PUBLICAPI)
-    def create_or_update(self, data: Data) -> Data:
+    def create_or_update(self, data: Union[Data,
```

Comment:
Let's wait to update the function interface. I have a spec on the "promote" experience for registry GA.

Reply:
Could you please elaborate on what you meant by "wait to update the function interface"? Do I need to wait for something else before making this update?

Reply:
I reversed the change to the create_or_update function interface